package.json

```
@@ -1,24 +1,27 @@
 {
   "license": "CC0-1.0",
   "engines": {
-    "node": ">=12",
+    "node": ">=18",
     "npm": "6.x.x"
   },
   "scripts": {
     "lint": "eslint . --ignore-pattern public --ignore-pattern vendor --ignore-pattern app/assets/builds",
-    "build": "webpack"
+    "build": "webpack --config webpack.config.js",
```

I would assume the `--config` value here would be default anyways and not necessary to specify?

```suggestion
    "build": "webpack",
```

---

updated in https://github.com/18F/identity-dashboard/commit/594bc2dafc173305e4c2b377dcf9ad97baabc8e9